scripts/io_gen.py

```diff
@@ -11,6 +11,7 @@
     'Span',
     'CodeActionKind',
     'AlsReferenceKind',
+    'AlsDisplayTypeHierarchyPolicy',
```

Comment:
(if you agree with changing the name of the flag in json, this should be changed accordingly - same for the Ada code)

Reply:
Done